### ISSUE TITLE
`save`: bug with torchvision

### ISSUE THREAD

**Issue by aguschin** [iterative/mlem#572]
The problem that causes bug: `weights` is Enum, defined as `strenum`, but its elements have a different type.

```
import pickle

from torchvision.models import ResNet50_Weights


def main():
    w = ResNet50_Weights.DEFAULT

    kek = pickle.dumps(w)
    pickle.loads(kek)


if __name__ == '__main__':
    main()
```

**Comment by mike0sv** [comment]
Can try when new tv version releases https://github.com/pytorch/vision/issues/7099

**Comment by aguschin** [comment]
Just checked, we're still waiting for a new release to check it and close if fixed... It was merged on 18.01.2023

**Comment by aguschin** [comment]
Works, cool! Closing.
